builtin/game/chatcommands.lua

```diff
@@ -576,14 +584,26 @@ local function handle_give_command(cmd, giver, receiver, stackstring)
 	-- The actual item stack string may be different from what the "giver"
 	-- entered (e.g. big numbers are always interpreted as 2^16-1).
 	stackstring = itemstack:to_string()
+	-- Air is not that useful as an item, but it also does no real harm.
+	-- Thus we do not forbid giving it, but we DO have a snarky comment. ;-)
+	local you_hacker = itemstack:get_name() == "air"
```

Comment:
Air and other airlike nodes may be given to players in different realms to breathe they don't really cause problems - IMO this check is not required.

Reply:
You have no sense of humour. :-(
Anyway, has this been actually used/abused on servers? Just curious. :D

Reply:
I'm trying to keep the quality of the code. Rare Easter eggs may be fine but Chuck Norris jokes or memes certainly belong somewhere else.
When I did some testing with the spacesuit and the moon realm generator, I also tried to cut a base into a hill and filling it with regular air to breathe - worked. Air has a low potential for (ab)use, which is probably also why nothing about is heard.

Reply:
Note that in the case of `air`, I merely adapted an old easter egg, I did not invent it.
I will remove it if enough core devs believe it must be wiped out.

Reply:
I prefer removing the 'you hacker' check and 'you hacker' messages, as i stated above.